### PR TITLE
[docs] Fix for duplicate titles and description 

### PR DESCRIPTION
**Pull request by aishwarya24**

### CODE REVIEW

File: docs/content/preview/develop/common-patterns/timeseries/ordering-by-entity.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ menu:
 type: docs
 ---
 
-In a time series data model, to enforce that all data for an entity stays together while maintaining the timestamp-based ordering, you have to distribute the data by the entity and order it by time.
+In a time-series data model, to enforce that all data for an entity stays together while maintaining the timestamp-based ordering, you have to distribute the data by the entity and order it by time.
```
Comment (ddhodge) [inline docs/content/preview/develop/common-patterns/timeseries/ordering-by-entity.md:15]: https://en.wikipedia.org/wiki/Time_series_database no hyphen it seems

File: docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Grammar Diagrams
+title: Grammar Diagrams YSQL
```
Comment (ddhodge) [inline docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md:2]: Revert all changes - pretty sure this is an auto generated file, and we just have to live with it. Anyhow, don't fix in this PR

**Review by ddhodge — APPROVED** [review]
Couple comments

File: docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md:2]: yes was unsure about this file. Okay let me revert it.

**Comment by aishwarya24** [comment]
Closing this one and creating another PR because I don't want to mess around reverting that auto-generated file. @ddhodge

**Pull request closed without merging**